### ISSUE TITLE
Web - Search Screen - JAWS is Reading Extra "Button"

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#107]
I found my screen reader JAWS now reads "Search submit button button",  for the newly added search button which is confusing.

I think that is because you've added the word "submit button" to the ARIA-LABEL and JAWS adds one because the element is a button. If you change the ARIA-LABEL to be "Search" or remove the ARIA and only use the text for the BUTTON, you will be fine.

Please note I only capitalized the ARIA text, so it would standout.

**Comment by KyleLivingstonDOL** [comment]
ZTI, can you please address?

**Comment by Abychalil** [comment]
Addressed this issue. Please retest and verify whether it meets the required 508 standards.
@jfc3-dol

**Comment by jfc3-dol** [comment]
The text "Search" is now a heading level one (h1) as expected, along with the results page names being a heading level two (h2) now too.

I'm wondering why there is a second heading level one (h1) for the text 10 search results for "text"? I would expect that to be a heading level two (h2) and then the results headings to be a heading level three (h3).

**Comment by jfc3-dol** [comment]
@Abychalil 
JAWS is now only reading the Search button as "Search button". It does not have read "Button" twice anymore.
I'm going to close this issue.

I did notice that when using IE 11, I can not submit a search the page only sits there. So I'm going to look to see if there is already an issue open for that and if not open a new one.
